go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/cloudfoundry-attic/jibber_jabber v0.0.0-20151120183258-bcc4c8345a21
 	github.com/cloudfoundry/jibber_jabber v0.0.0-20151120183258-bcc4c8345a21 // indirect
 	github.com/docker/cli v0.0.0-20200303162255-7d407207c304 // indirect
-	github.com/docker/docker v17.12.0-ce-rc1.0.20200916142827-bd33bbf0497b+incompatible
```

Comment:
is there a PR upstream that we could comment here to remove the replace after the PR gets merged ?

Reply:
Alternatively one could upgrade everything to the Docker 20.10 API, but that is much more intrusive and big

Reply:
The patch was: https://github.com/moby/moby/commit/c3a0a3744636069f43197eb18245aaae89f568e5